src/pages/signing_official_console/SigningOfficialTable.jsx

```diff
@@ -426,10 +444,10 @@ export default function SigningOfficialTable(props) {
       />
       <LibraryCardFormModal
         showModal={showModal}
-        createOnClick={(card) => issueLibraryCard(card, researchers)}
+        createOnClick={(cards) => issueLibraryCards(cards, researchers)}
         closeModal={() => setShowModal(false)}
         card={selectedCard}
-        users={onlyResearchersWithoutCardFilter(researchers)}
+        users={researchers.filter(onlyResearchersWithoutCardFilter)}
```

Comment:
This line specifically addresses https://broadworkbench.atlassian.net/browse/DT-1851